config/routes.config.js

```diff
@@ -5,12 +5,14 @@ const securityQuestionUrls = [
   '/login/questions/prison',
   '/login/questions/dateOfResidence',
   '/login/questions/bankruptcy',
-  '/login/questions/trillium',
   '/login/questions/bank',
   '/login/questions/taxReturn',
   '/login/questions/rrsp',
   '/login/questions/tfsa',
   '/login/questions/ccb',
+
+  // Skip URL
+  '/checkAnswers',
```

Comment:
We don't need this as it's already in our routes -> https://github.com/cds-snc/cra-claim-tax-benefits/blob/master/config/routes.config.js#L52

Reply:
this will throw an error on the security page saying it's not a possible option if removed though